packages/arb-token-bridge-ui/src/util/wagmi/setup.ts

```diff
@@ -154,7 +159,11 @@ export function getProps(targetChainKey: string | null) {
     ...wallets,
     {
       groupName: 'More',
-      wallets: [trustWallet({ chains, projectId })]
+      wallets: [
+        trustWallet({ chains, projectId }),
+        rabbyWallet({ chains }),
+        injectedWallet({ chains })
```

Comment:
injected wallet is already included in the `wallets` array

it will show up if `window.ethereum` is present, for example when user has Frame

with this change we have two of them

rainbowkit's docs stated that it is automatically hidden if not an option
> This is a fallback wallet option designed for scenarios where window.ethereum exists but hasn’t been provided by another wallet in the list. This wallet will automatically hide itself from the list when the fallback is not necessary or if there is no injected wallet available.
<img width="826" alt="image" src="https://github.com/user-attachments/assets/ef4d1e86-313e-4693-89a8-34f1a3aebf00" />